.travis.yml

```diff
@@ -1,13 +1,18 @@
-sudo: required
-services:
-    - docker
 language: java
-jdk:
-    - oraclejdk8
-script: mvn verify
+matrix:
+    include:
+        - os: linux
+          jdk: oraclejdk8
+          services:
+              - docker
+        - os: osx
+          osx_image: xcode8.2
+script:
```

Comment:
What about `mvn $goal` and have a `goal` environment variable defined on both os ?